index.d.ts

```diff
@@ -8,15 +8,10 @@
 //   Michal Adamczyk <https://github.com/mradamczyk>
 //   Marvin Hagemeister <https://github.com/marvinhagemeister>
 
-export type ClassValue = string | number | ClassDictionary | ClassArray | undefined | null | boolean;
+export type Value = string | number | boolean | undefined | null;
+export type Mapping = { [key: string]: Value };
+export type Argument = Value | Mapping | Argument[];
```

Comment:
I don't think we necessarily needed to `export` these... but, if it's useful,  why not.